### ISSUE TITLE
Error running test.py and train.py

### ISSUE THREAD

**Issue by matheusfalcaopinto** [WongKinYiu/yolor#184]
I'm running in google colab:

`!python test.py --data data/coco.yaml --img 1280 --batch 32 --conf 0.001 --iou 0.65 --device 0 --cfg cfg/yolor_p6.cfg --weights yolor_p6.pt --name yolor_p6_val`

or

`!python detect.py --source inference/images/horses.jpg --cfg cfg/yolor_p6.cfg --weights yolor_p6.pt --conf 0.25 --img-size 1280 --device 0`

or

`python train.py --batch-size 8 --img 1280 1280 --data coco.yaml --cfg cfg/yolor_p6.cfg --weights '' --device 0 --name yolor_p6 --hyp hyp.scratch.1280.yaml --epochs 300
`

and I get:

`_pickle.UnpicklingError: invalid load key, '<'.`

Any suggestions? Thanks!

**Comment by Faisalsouz** [comment]
Having same problem @matheusfalcaopinto  have you found solution or anybody else can suggest where exactly problem lies.Thanks

**Comment by JoseIgnDiazVilla** [comment]
I'm getting the same error. Tho I suspect it's from a version mismatch between torch and some other package.

**Comment by matheusfalcaopinto** [comment]
> I'm getting the same error. Tho I suspect it's from a version mismatch between torch and some other package.

Yes, but here the more I tried to change the version of packages, more errors I get haha

**Comment by ZBC043** [comment]
> I found a solution: the reason of the error is that the file pretrain.sh bash is not downloading the weights properly. As script is trying to crease cookies and then supply it API key. The easy way is to just go here "https://drive.google.com/uc?export=download&id=1Tdn3yqpZ79X7R1Ql0zNlNScB1Dv9Fp76" and download weight manually.


Hi, it seems like I cannot open this link, can you give me some help?
